src/main/java/org/jitsi_modified/impl/neomedia/codec/video/vp8/DePacketizer.java

```diff
@@ -146,7 +162,54 @@ public static int getTemporalLayerIndex(byte[] buf, int off, int len)
                 return -1;
             }
 
-            return (buf[off + sz - 1] & 0xc0) >> 6;
+            return (byte) (buf[off + sz - 1] & 0xFF);
```

Comment:
What does the `& 0xFF` do here? This is a byte.

Reply:
This makes sure that if regular value of the byte is greater than 127, then it won't change to negative values when converting to integer.

Reply:
But doesn't it convert to negative anyway when you cast to `byte`?

Reply:
True, fixed